mcs/class/referencesource/mscorlib/system/convert.cs

```diff
@@ -2472,7 +2476,7 @@ public static decimal ToDecimal(DateTime value)
         [SecurityCritical]
         private static unsafe Int32 FromBase64_Decode(Char* startInputPtr, Int32 inputLength, Byte* startDestPtr, Int32 destLength) {
 
-            // You may find this method weird to look at. It�s written for performance, not aesthetics.
+            // You may find this method weird to look at. It’s written for performance, not aesthetics.
```

Comment:
why this change?

Reply:
I did not change it - seems like IDE did it (vscode or rider) 🙂